docs/components/header.css

```diff
@@ -38,9 +38,8 @@
 }
 
 .logoImage {
-  padding-top: 20px;
-  padding-left: 10px;
-  padding-right: 10px;
+  padding: 20px 10px 0 10px !important;
```

Comment:
Never use `!important` unless you really really REALLY have no choice. The only time that's the case is when you are fighting external CSS such as blueprint.

See my other comment about how to fix the img 100% width issue

Reply:
Yes have done that.

Reply:
Can we remove these importants now?

Reply:
Oops yes, sorry